hudi-client/src/main/java/org/apache/hudi/index/bloom/HoodieGlobalBloomIndex.java

```diff
@@ -125,6 +125,9 @@ public HoodieGlobalBloomIndex(HoodieWriteConfig config) {
           // Create an empty record to delete the record in the old partition
           HoodieRecord<T> emptyRecord = new HoodieRecord(recordLocationHoodieKeyPair.get()._2,
               new EmptyHoodieRecordPayload());
+          emptyRecord.unseal();
+          emptyRecord.setCurrentLocation(recordLocationHoodieKeyPair.get()._1());
+          emptyRecord.seal();
```

Comment:
It seems that we don't need to unseal() as the sealed is `false` when using 
https://github.com/apache/hudi/blob/574dcf920c4677513f6fdc8d441bb42827afa5a2/hudi-common/src/main/java/org/apache/hudi/common/model/HoodieRecord.java#L67-L73